packages/x-date-pickers-pro/src/DateTimeRangePicker/DateTimeRangePickerTimeWrapper.tsx

```diff
@@ -48,10 +47,7 @@ function DateTimeRangePickerTimeWrapper<
     Omit<BaseClockProps<TView>, 'value' | 'defaultValue' | 'onChange'>,
     'views'
   >,
->(
-  props: DateTimeRangePickerTimeWrapperProps<TView, TComponentProps>,
-  ref: React.Ref<HTMLDivElement>,
```

Comment:
Seems like a mistake, it was not receiving a ref and wasn't wrapped in `forwardRef`. 🤷